source/slang/slang-ir-glsl-legalize.cpp

```diff
@@ -259,11 +259,20 @@ List<IRInst*> ScalarizedVal::leafAddresses()
 
 struct GLSLLegalizationContext
 {
-    Session*                session;
-    GLSLExtensionTracker*   glslExtensionTracker;
-    DiagnosticSink*         sink;
-    Stage                   stage;
-    IRFunc*                 entryPointFunc;
+    Session*                   session;
+    GLSLExtensionTracker*      glslExtensionTracker;
+    DiagnosticSink*            sink;
+    Stage                      stage;
+    IRFunc*                    entryPointFunc;
+    
+    //Dictionary<ResourceType, Dictionary<Space, BindingIndex>>
```

Comment:
Add more description to the comment as it currently looks too much like a left over debug code.